hack/generator/pkg/astbuilder/builder.go

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astbuilder
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+)
+
+// CheckErrorAndReturn checks if the err is non-nil, and if it is returns.
```

Comment:
It would be useful to have the output syntax that these correspond to in the doc comments.

Reply:
Added